includes/admin/load.php

```diff
@@ -77,7 +77,17 @@ function perflab_render_settings_page() {
  * @param string $hook_suffix The current admin page.
  */
 function perflab_admin_pointer( $hook_suffix ) {
```

Comment:
```
// Do not show admin pointer in multisite Network admin or User admin UI.
if ( is_network_admin() || is_user_admin() ) {
	return;
}
```

Let's check this condition at the top of the function, as the changes do not need to run on multisite.

Reply:
Edit thanks :)